CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.25.0
 
 * Allow a custom password to be set for SCRAM-SHA-512 users by referencing a secret in the `KafkaUser` resource
+* Add support for `tls-noop` authentication to User Operator to allow management of ACLs and Quotas for TLS users with user certificates generated externally 
```

Comment:
I don't know if it's useful for anyone else, but I would appreciate the PR number against these descriptions to check the work.